pkg/pidfile/pidfile.go

```diff
@@ -0,0 +1,44 @@
+package pidfile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+type PidFile struct {
```

Comment:
@ahmetalpbalkan I played a bit around with it, but it looks like you'll get something like

```go
file = (*PidFile)(&path)
```

and

```go
os.Remove(string(file));
```

This looks a bit ugly to me... Don't you think?